lib/services/metadata.test.js

```diff
@@ -69,13 +69,53 @@ describe(_.startCase(filename), function () {
       db.patchMeta.returns(Promise.resolve());
       db.getMeta.returns(Promise.resolve());
 
-      return fn('domain.com/_pages/id/meta', {})
+      return fn('domain.com/_pages/id/meta', {}, {})
```

Comment:
you set the default for the 2nd and 3rd argument in the function, they can be removed

Reply:
I sent these params in here so anyone who have a look at the tests knows that it receives more than one param